devito/core/gpu_openacc.py

```diff
@@ -2,15 +2,15 @@
 
 import cgen as c
 
-from devito.core.gpu_openmp import (DeviceOpenMPNoopOperator, DeviceOpenMPIteration,
-                                    DeviceOmpizer, DeviceOpenMPDataManager)
-from devito.exceptions import InvalidOperator
+from devito.core.gpu_openmp import (DeviceOpenMPNoopOperator, DeviceOpenMPOperator,
```

Comment:
How about renaming gpu_openmp and it's members as it's used for both OpenMP and OpenACC.

Reply:
yes. There is a TODO somewhere (I think in gpu_openacc) which explains that some major renaming (actually, refactoring) is necessary. This was like this before this PR, not sure if we really want to do that here or afterwards